stac_fastapi/elasticsearch/setup.py

```diff
@@ -29,6 +29,7 @@
         "requests",
         "ciso8601",
         "overrides",
+        "starlette",
```

Comment:
we have use some classes out of this package, but it was not explicitly deps

Reply:
I recently had to add this for a fresh pgstac stac-fastapi backend that we're testing at work